src/fragment-components/tiles/a-clickable-tile.tsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import {
+	TextInput,
+	Checkbox,
+	ClickableTile
+} from 'carbon-components-react';
+import { AComponent } from '../a-component';
+import { TileMorphism } from './tile-morphism';
+import { css } from 'emotion';
+import { ComponentCssClassSelector } from '../../components/css-class-selector';
+import { ComponentInfo } from '..';
+import image from '../../assets/component-icons/tile-clickable.svg';
+import {
+	angularClassNamesFromComponentObj,
+	nameStringToVariableString,
+	reactClassNamesFromComponentObj
+} from '../../utils/fragment-tools';
+
+export const AClickableTileStyleUI = ({ selectedComponent, setComponent }: any) => {
+	return <>
+		<TileMorphism component={selectedComponent} setComponent={setComponent} />
+		<Checkbox
+			labelText='Light theme'
+			id='theme-select'
+			checked={selectedComponent.light}
+			onChange={(checked: any) => {
+				setComponent({
+					...selectedComponent,
+					light: checked
+				});
+			}}
+		/>
+		<Checkbox
+			labelText='Disabled'
+			id='disabled'
```

Comment:
I use the same ids as well for my disabled fields, I wonder if it makes sense to make these input field ids unique for the sake of accessibility. Might be fine for now though just something to think about down the line.

Reply:
Not necessary, we only render the input fields for the component we select